### DISCUSSION TITLE
Extension to Palmgren Silver Images

### DISCUSSION THREAD

**Discussion by abadgerw** [neuropoly/axondeepseg#843] [Help needed]
I was wondering whether it would be possible to extend this platform to detecting axons identified using Palmgren silver staining and light microscopy?

**Comment by mathieuboudreau** [comment]
@abadgerw do you have sample images you could share?

**Comment by mathieuboudreau** [comment]
If not on github, you can email them to us at axondeepseg (AT) googlegroups (DOT) com

**Comment by abadgerw** [comment]
@mathieuboudreau Just sent over an email. Do not hesitate to let me know if you need anything else or have any questions.

**Comment by mathieuboudreau** [comment]
Converting this to use discussions
